libs/internal/include/launchdarkly/data_model/segment.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <boost/json/value.hpp>
```

Comment:
Could we group these into internal dependencies, boost dependencies, and std deps?